rep-2014.rst

```diff
@@ -0,0 +1,243 @@
+REP: 2014
+Title: Benchmarking performance in ROS 2
+Author: Víctor Mayoral-Vilches <victor@accelerationrobotics.com>, Ingo Lütkebohle <Ingo.Luetkebohle@de.bosch.com>, Christophe Bédard <bedard.christophe@gmail.com>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 29-Sept-2022
+Post-History: 05-Nov-2022
+
+
+Abstract
+========
+
+This REP describes some principles and guidelines for benchmarking performance in ROS 2.
+
+
+Motivation
+==========
+
+Benchmarking is the act of running a computer program with a known workload to assess the program's relative performance. In the context of ROS 2, performance information can help roboticists design more efficient robotic systems and select the right hardware for their robotic application. It can also help understand the trade-offs between different algorithms that implement the same capability, and help them choose the best approach for their use case. Performance data can also be used to compare different versions of ROS 2 and to identify regressions. Finally, performance information can be used to help prioritize future development efforts.
```

Comment:
One general point: Each sentence should have its own line. It makes it easier to give feedback, and it makes the diff smaller in future revisions.

Reply:
Noted, we'll look into this in the next iteration of reviews.